File: Volume 11 (1100 - 1199)/1158/1158.c
#include <stdio.h>
 
int main() {
    int x, y, n, i, k, total;
    scanf("%d", &n);
    while (n--) {
        total = 0;
        scanf("%d %d", &x, &y);
        for (i = x, k = 0; k < y; i++) {
            if (i % 2 != 0) {
                total += i;
                k++;
            }
        }
        printf("%d\n", total);
    }
}

File: Volume 11 (1100 - 1199)/1158/1158.py
for i in range(0, int(input())):
    total = 0
    s = [int(j) for j in input().split(' ')]
    if (s[0] % 2 == 0):
        total += s[0] + 1
        s[0] += 1
        for k in range(1,s[1]):
            s[0] += 2
            total += s[0]
    else:
        total += s[0]
        for k in range(1,s[1]):
            s[0] += 2
            total += s[0]
    print(total)